README.md

```diff
@@ -4,4 +4,4 @@
 
```

Comment:
Please, kindly update this section of your readme by adding the following headers:

1. Contribution header: Kindly add all the contents in the contributing header, as you've removed some contents from it. You can refer to the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) to add the missing contents.
2. Show your support header.
 
The arrangement should follow that used in the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) to keep consistency to make your repo appear professional.